src/components/Modal/index.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react';
```

Comment:
the focus should stay in and cycle through the modal’s content and not to underlying page